Animations/Animation.js

```diff
@@ -0,0 +1,126 @@
+// 'use strict';
```

Comment:
Uncomment line one and add a newline between it and the React require statement.